package.json

```diff
@@ -17,22 +17,21 @@
   "scripts": {
     "build": "lerna run build",
     "changelog": "auto-changelog --remote upstream --commit-limit false",
-    "cnp": "NODE_ENV=test packages/create-project/bin/create-neutrino-project.js $(yarn random)",
     "link:all": "lerna exec yarn link",
     "lint": "node packages/neutrino/bin/neutrino lint",
     "precommit": "lint-staged",
     "random": "node -e \"process.stdout.write(require('crypto').randomBytes(8).toString('hex'))\"",
-    "release": "yarn build && lerna publish --force-publish=*",
+    "release": "yarn build && NEUTRINO_RELEASE=true lerna publish --force-publish=*",
```

Comment:
The pre-existing `yarn build` doesn't actually do anything as far as I can tell? Should we remove it (either now or at another time)?